lastuser_oauth/mailclient.py

```diff
@@ -5,24 +5,62 @@
 from flask.ext.mail import Mail, Message
 from baseframe import _
 from html2text import html2text
+from uuid import uuid4
+import pystache
 
 mail = Mail()
 
 
-def send_email_verify_link(useremail):
+def verify_email_template(template, reset=False):
+    """
+    Verifies that the template is valid and contains a tag for email
+    """
+    try:
+        random_email = unicode(uuid4())
+        if random_email in pystache.render(template, {'email': random_email}):
```

Comment:
Yup. Was initially checking for multiple fields. This can be squashed to one line now.